modules/logging-loki-cli-install.adoc

```diff
@@ -37,9 +69,186 @@ spec:
+
+. Apply the Operator Group object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f olo-og.yaml.yaml
+----
++
+--
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+<1> You must specify the `openshift-logging` namespace.
+--
+
+. Apply the Subscription object YAML configuration that follows:
++
+[source,terminal]
+----
+$ oc apply -f olo-sub.yaml
+----
++
+--
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable # <2>
+  name: cluster-logging
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+--
+
+. Apply the instance object YAML configuration that follows:
+[IMPORTANT]
+====
+It is not possible to change the number `1x` for the deployment size.
+====
++
+[source,terminal]
+----
+$ oc apply-f logging-loki.yaml
+----
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 # <4>
+      type: s3 # <5>
+      credentialMode: # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify the `openshift-logging` namespace.
+<3> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<4> Specify the name of your log store secret.
+<5> Specify the corresponding storage type.
+<6> Optional field, logging 5.9 and later. Supported user configured values are as follows: static is the default authentication mode available for all supported object storage types using credentials stored in a Secret. token for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types. token-cco is the default value when Loki is running on managed STS mode and using CCO on STS/WIF clusters.
```

Comment:
What are the actual variables here? I think that one is `static`. Because there's so much text I think that it's difficult to pinpoint which ones users can use. They should probably go in backticks.